src/MongoDB/ReadPreference.c

```diff
@@ -46,21 +46,22 @@ PHONGO_API zend_class_entry *php_phongo_readpreference_ce;
 
 zend_object_handlers php_phongo_handler_readpreference;
 
-/* {{{ proto void ReadPreference::__construct(integer $mode[, array $tagSets = array()])
+/* {{{ proto void ReadPreference::__construct(integer $mode[, array $tagSets = array()[, integer $maxStalenessMS]])
```

Comment:
I would much rather see this being an options array, as it is possible that other options might be added in the future.